### ISSUE TITLE
Ubuntu 17.10 deb package depends on the wrong libicu package

### ISSUE THREAD

**Issue by rakeshsinghranchi** [dotnet/core-setup#4494]
_From @Genbox on July 20, 2018 16:42_

When installing .NET Core 2.0.9 runtime (dotnet-runtime-2.0.9) on Ubuntu 17.10, it depends on libicu55 instead of libicu57 as it should. It is correctly stated on the [documentation page](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x):

> libicu55 (for 16.x)
> libicu57 (for 17.x)

However, when you look in the [packages file](https://packages.microsoft.com/ubuntu/17.10/prod/dists/artful/main/binary-amd64/Packages) from the Microsoft repository, it depends on libicu55 instead of libicu57:

> Package: dotnet-runtime-2.0.9
> Depends: libc6 (>= 2.14), libcurl3 (>= 7.16.2), libgcc1 (>= 1:3.0), libgssapi-krb5-2 (>= 1.10+dfsg~), liblttng-ust0 (>= 2.5.0), libstdc++6 (>= 4.8), libunwind8, libuuid1 (>= 2.16), zlib1g (>= 1:1.1.4), **libicu55**, libssl1.0.0, dotnet-hostfxr-2.0.9

The dotnet-runtime-2.0.7 has the correct dependency (libicu57)

Semi related: #20562

_Copied from original issue: dotnet/corefx#31224_

**Comment by rakeshsinghranchi** [comment]
_From @Genbox on July 20, 2018 16:43_

Mentioning @lordmike

**Comment by rakeshsinghranchi** [comment]
_From @joshfree on July 20, 2018 19:46_

Thanks for reporting this @Genbox.  @tarekgh could you PTAL?

**Comment by rakeshsinghranchi** [comment]
_From @tarekgh on July 20, 2018 22:50_

@joshfree this is SDK packaging issue. I think @rakeshsinghranchi may be able to help here.

**Comment by rakeshsinghranchi** [comment]
_From @LordMike on August 9, 2018 22:51_

Is there any update on this? A newly deployed Ubuntu 17.10 on AWS EC2 still cannot install dotnet-runtime-2.0.9. I've had to re-use my modified dpkg package from 3 weeks ago, where I altered the dependency in question.

**Comment by rakeshsinghranchi** [comment]
_From @tarekgh on August 9, 2018 22:54_

@rakeshsinghranchi could you please help with this issue?

**Comment by rakeshsinghranchi** [comment]
@leecow , can you please confirm if correct rutime package are available in the feed for ubuntu 17.10 ?

**Comment by leecow** [comment]
Yes, 2.0.9 is available on the 17.10 feed. 

```
root@fecc1bc53856:/# cat /etc/apt/sources.list.d/microsoft-prod.list
deb [arch=amd64] https://packages.microsoft.com/ubuntu/17.10/prod artful main

root@fecc1bc53856:/# cat /etc/os-release
NAME="Ubuntu"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 17.10"
VERSION_ID="17.10"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful

root@fecc1bc53856:/# apt search dotnet-runtime-2.0.9
Sorting... Done
Full Text Search... Done
dotnet-runtime-2.0.9/artful 2.0.9-1 amd64
  Microsoft .NET Core Runtime - 2.0.9 Microsoft.NETCore.App 2.0.9
```

**Comment by leecow** [comment]
2.0.9 runtime package has been updated and tested. 17.10 feed should be good to go now. 

Note that [17.10 reached end-of-life in July and 2.0.9](2.0-supported-os.md) is the last .NET Core update which will be delivered for 17.10.
